src/main/java/HookKiller/server/service/NoticeService.java

```diff
@@ -0,0 +1,90 @@
+package HookKiller.server.service;
+
+import HookKiller.server.common.type.LanguageType;
+import HookKiller.server.notice.dto.NoticeArticleDto;
+import HookKiller.server.notice.dto.NoticeContentDto;
+import HookKiller.server.notice.entity.NoticeArticle;
+import HookKiller.server.notice.entity.NoticeContent;
+import HookKiller.server.notice.exception.NoticeArticleNotFoundException;
+import HookKiller.server.repository.NoticeArticleRepository;
+import HookKiller.server.repository.NoticeContentRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class NoticeService {
+
+    private final NoticeArticleRepository noticeArticleRepository;
+    private final NoticeContentRepository noticeContentRepository;
+
+    @Transactional
+    public void saveNotice(NoticeArticleDto articleDto, NoticeContentDto contentDto) {
+        NoticeArticle noticeArticle = NoticeArticle.builder()
+                .id(articleDto.getId())
+                .build();
+        NoticeContent noticeContent = NoticeContent.builder()
+                        .id(contentDto.getId())
+                                .build();
+        noticeArticleRepository.save(noticeArticle);
+        noticeContentRepository.save(noticeContent);
+    }
+
+    @Transactional(readOnly = true)
+    public List<NoticeArticleDto> getNoticeList() {
+
+        List<NoticeArticle> all = noticeArticleRepository.findAll();
+        List<NoticeArticleDto> noticeArticleDtoList = new ArrayList<>();
+
+        for (NoticeArticle noticeArticle : all) {
+            NoticeArticleDto articleDto = NoticeArticleDto.builder()
+                    .id(noticeArticle.getId())
+                    .language(noticeArticle.getLanguage())
+                    .status(noticeArticle.getStatus())
+                    .createdAt(noticeArticle.getCreatedAt())
+                    .createdUser(noticeArticle.getCreatedUser())
+                    .updatedAt(noticeArticle.getUpdatedAt())
+                    .updatedUser(noticeArticle.getUpdatedUser())
+                    .build();
+            noticeArticleDtoList.add(articleDto);
+        }
+        return noticeArticleDtoList;
+    }
+
+    @Transactional
+    public NoticeContentDto getNotice(Long id) {
+        NoticeContent noticeContent = noticeContentRepository.findById(id).orElseThrow(() -> NoticeArticleNotFoundException.EXCEPTION);
+
+        return NoticeContentDto.builder()
+                .id(noticeContent.getId())
+                .language(noticeContent.getLanguage())
+                .title(noticeContent.getTitle())
+                .content(noticeContent.getContent())
+                .build();
+    }
```

Comment:
이 코드는 나중에 또 쓸 수 있으니
`NoticeContentDto`에 `from` 메소드를 만들어보는 것은 어떨까요?
```suggestion
    public NoticeContentDto getNotice(Long id) {
        NoticeContent noticeContent = noticeContentRepository.findById(id).orElseThrow(() -> NoticeArticleNotFoundException.EXCEPTION);

        return NoticeContentDto.from(noticeContent);
    }
```
처럼 나오도록 한번 만들어보세요!